test/system/700-play.bats

```diff
@@ -1035,3 +1035,72 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman ply with multi automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile1 <<EOF
+FROM $IMAGE
+RUN mkdir /test1 /test_same
+RUN touch /test1/a /test1/b /test1/c
+RUN echo "I am from test1 image" > /test_same/hello_world
```

Comment:
please chain them via `&& \` instead of desperate RUN instructions

Reply:
OK. Will update this.